scripts/start-setupForgeApiMods

```diff
@@ -77,7 +77,7 @@ retrieveVersionTypeNumber(){
 }
 
 modFileByProjectID(){
-  project_id=$(echo $1 | tr -d '"')
+  project_id=$(echo "$1" | tr -d '"')
```

Comment:
Could be just this, but can save it for a future PR
```suggestion
  project_id="$1"
```

Reply:
Sorry was at the gym, yes I saw the trim. I almost cut it out. Good call though! Thanks for the merge!